### PR TITLE
Fix component versions for release

### PR DESCRIPTION
**Pull request by NisalaNiroshana**
## Purpose
Update version in dashboard components.

### CODE REVIEW

File: features/org.wso2.carbon.dashboards.portal.feature/src/main/resources/portal/extensions/widgets/Publisher/Publisher.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...e"in r&&(r.writable=!0),Object.defineProperty(t,r.key,r)}}return function(e,n,r){return n&&t(e.prototype,n),r&&t(e,r),e}}(),a=n(6),s=function(t){return t&&t.__esModule?t:{default:t}}(a),c=function(t){function e(t){r(this,e);var n=o(this,(e.__proto__||Object.getPrototypeOf(e)).call(this,t));return n.getDashboardAPI=n.getDashboardAPI.bind(n),n.subscribe=n.subscribe.bind(n),n}return i(e,t),u(e,[{key:"subscribe",value:function(t){this.props.glEventHub.on(this.props.id,t)}},{key:"publish",value:function(t){var e=this.props.id;this.props.glEventHub.emit(e,t)}},{key:"getDashboardAPI",value:function(){function t(){return""===window.location.hash||"#"===window.location.hash?{}:JSON.parse(window.location.hash.substr(1))}function e(t){window.location.hash=JSON.stringify(t)}function n(){var e=t();return e.hasOwnProperty(o.props.id)?e[o.props.id]:{}}function r(n){var r=t();r[o.props.id]=n,e(r)}var o=this;return{state:{get:function(t){var e=n();return e.hasOwnProperty(t)?e[t]:null},set:function(t,e){var o=n();o[t]=e,r(o)}}}}},{key:"render",value:function(){var t={padding:"30px 15px 15px 15px"};return s.default.createElement("div",{style:t},this.renderWidget())}}]),e}(a.Component);e.default=c}]);
```
Comment (TanyaM) [inline features/org.wso2.carbon.dashboards.portal.feature/src/main/resources/portal/extensions/widgets/Publisher/Publisher.js]: What does this file do? And why is it minified ?

File: samples/Publisher/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "react-dom": "^15.6.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "webpack -p",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject"
+  },
+  "devDependencies": {
+    "ajv": "^5.2.2",
```
Comment (TanyaM) [inline samples/Publisher/package.json]: Check if this is used, if not remove

File: samples/Publisher/src/Publisher.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    publishMsg(event) {
+        event.preventDefault();
+        if (this.input.value) {
+            this.state.inputVal = '';
+            this.publishedMsgSet.add(this.input.value);
+            super.publish(this.input.value);
+            this.setState({publishedMsg: this.input.value});
+        } else {
+            alert('No content to publish !');
```
Comment (TanyaM) [inline samples/Publisher/src/Publisher.jsx]: Remove alerts

File: features/org.wso2.carbon.dashboards.portal.feature/src/main/resources/portal/extensions/widgets/Publisher/Publisher.js (hunk 1 above)
Comment (NisalaNiroshana) [reply features/org.wso2.carbon.dashboards.portal.feature/src/main/resources/portal/extensions/widgets/Publisher/Publisher.js]: This is added to portal app until we properly handle the sample as discussed offline.

**Pull request merged**